src/app/feed/feed-card/feed-card.component.spec.ts

```diff
@@ -46,7 +47,106 @@ describe('Component: FeedCard', () => {
 		});
 	});
 	it('should create an instance', () => {
-		const component = TestBed.createComponent(FeedCardComponent);
+		const fixture = TestBed.createComponent(FeedCardComponent);
+		const component = fixture.debugElement.componentInstance;
 		expect(component).toBeTruthy();
 	});
+
+	it('should set inviewport to true on calling inview(true)', async(() => {
+		const fixture = TestBed.createComponent(FeedCardComponent);
+		const component = fixture.debugElement.componentInstance;
+		component.inview({ value: true });
+
+		expect(component.inviewport).toBe(true);
+	}));
+
+	it('should use feedItem', async(() => {
+		const fixture = TestBed.createComponent(FeedCardComponent);
+		const component = fixture.debugElement.componentInstance;
+
+		component.feedItem = {
+			images : [],
+			screen_name : 'fossasia',
+			user : {
+				name : 'testUser'
+			},
+			text : 'testText',
+			retweet_count : 20,
+			favourites_count : 25,
+			created_at : new Date(),
+			videos : []
+		};
+		component.ttt();
+		expect(component.datetime).toBe('now');
+		expect(component.itemText).toBe('testText');
+		expect(component.profileName).toBe('testUser');
+		expect(component.retweetCount).toBe('20');
+		expect(component.favoriteCount).toBe('25');
+		expect(component.profileURL).toBe('https://twitter.com/fossasia/');
+	}));
+
+	it('should have feedLinker component', async(() => {
+		const fixture = TestBed.createComponent(FeedLinkerStubComponent);
+		const component = fixture.debugElement.componentInstance;
+
+		expect(component).toBeTruthy();
+	}));
+
+	it('should have app-lazy-img component', async(() => {
+		const fixture = TestBed.createComponent(LazyImgStubComponent);
+		const component = fixture.debugElement.componentInstance;
+
+		expect(component).toBeTruthy();
+	}));
+
+	it('should emit on click', async(() => {
```

Comment:
Add better display name.

Reply:
Yeah I think `'should emit showLightBox EventEmitter on click'` will be better. What say?

Reply:
Yes it sounds good.